package.json

```diff
@@ -99,6 +98,7 @@
     "lint-staged": "^13.1.2",
     "react-test-renderer": "^18.2.0",
     "remark-gfm": "^3.0.1",
+    "sass": "^1.62.1",
```

Comment:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/60910665/234649593-d17c36f3-85a2-4a65-8259-65a9f52bf637.png">

https://stackoverflow.com/questions/55089432/is-node-sass-a-dev-or-a-production-dependency-on-react-projects